assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -292,18 +290,18 @@ che.infra.kubernetes.ingress.domain=
 # DEPRECATED - Do not change the value of this property otherwise the existing workspaces will loose data. Do not
 # set it on new installations.
 #
-# Defines Kubernetes namespace in which all workspaces will be created.
+# Defines {orch-namespace} in which all workspaces will be created.
```

Comment:
what is `{orch-namespace}` ? does it become `Project` on Openshift? Then I think this is wrong.

Reply:
Yes it does.